at.ssh-mitm.server.json

```diff
@@ -0,0 +1,138 @@
+{
+    "app-id": "org.flatpak.Hello",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
```

Comment:
```suggestion
    "runtime-version": "20.08",
```

Reply:
> it's also likely missing an appstream file.

I have added an appstream file to this pull request.